airbyte-integrations/connectors/source-pinterest/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 5cb7e5fe-38c2-11ec-8d3d-0242ac130003
-  dockerImageTag: 0.2.4
```

Comment:
Note for posterity if anyone is confused why this skipped a version - these metadata files were generated once a little while ago. They're currently not in use anywhere, so they may go stale, and that's okay. The place to update for the moment is still the [dockerfile](https://github.com/airbytehq/airbyte/pull/25548/files#diff-6eeb808f0899bd5167d03b761160d0ec6566f692829dc5317a5e935bcfdb7151R37) as has been done in this PR. 

Eventually we'll switch over to publish on merge, and that publish workflow will bump the dockerfile and the metadata version in sync. that's when the metadata files will start always being in sync (and we'll re run generation again to make sure that nothing is out of date before starting this process). 

Cc @bnchrch to check my work